server/index-node/src/resolver.rs

```diff
@@ -286,6 +287,9 @@ where
         let mut response: BTreeMap<String, q::Value> = BTreeMap::new();
         response.insert("features".to_string(), features);
         response.insert("errors".to_string(), errors);
+        if let Some(network_name) = network {
```

Comment:
Does it make sense to always return a `network` field, but `null` when `errors` is filled? 🤔 

From my understanding the way it is written in this PR, if the validation worked, the field exists, otherwise it isn't returned.